### ISSUE TITLE
Fix TaxoPress AI Synonyms not working properly 

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/TaxoPress#2013]
Should it be possible for synonyms to trigger suggestions in TaxoPress AI?

Right now, it's only for Auto Terms https://taxopress.com/docs/synonyms/

**Comment by ojopaul** [comment]
@stevejburge Do you mind breaking this request down?

**Comment by ojopaul** [comment]
@stevejburge To be sincere, I still don't get. Synonyms are not terms and TaxoPress AI screen display terms. So, how do you see this working?

**Comment by stevejburge** [comment]
@ojopaul I believe this request was for the "Suggest Existing Terms" feature so it could be a checkbox there

**Comment by ojopaul** [comment]
But isn't the feature already working with TaxoPress AI? @stevejburge

**Comment by stevejburge** [comment]
Do you mean if we enable the setting inside "Auto Terms"?

**Comment by ojopaul** [comment]
See here https://github.com/TaxoPress/TaxoPress/blob/development/modules/taxopress-ai/classes/TaxoPressAiAjax.php#L360 @stevejburge

**Comment by stevejburge** [comment]
@ojopaul Maybe let's change the approach. Is there any chance that this could be a separate setting under "TaxoPress AI"?

It will be good to separate the settings between TaxoPress AI and Auto Terms.

**Comment by ojopaul** [comment]
@stevejburge 
Let me rephrase this, 
1. TaxoPress AI is not using settings from Auto Terms or Auto Link
2. Linked Terms and Synonyms work automatically with TaxoPress AI Suggest existing tab(only) contrary to this issue that ask to support it (or can you confirm it's not working in your settings?)
3. If you feel it's okay to have a settings for such, maybe add an issue on that to clarify what we're working on?

**Comment by stevejburge** [comment]
@ojopaul OK, thanks. Let's keep it simple in this issue then.

In my testing, Synonyms weren't working with "Suggest Existing Terms"
